### PR TITLE
homework_2

### PR DESCRIPTION
**Pull request by viskosa**
I've done it)

### CODE REVIEW

File: js-core/homeworks/homework-2/src/main.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+ ' +
+  frameworks[1] +
+  Math.floor(frameworks[0]) +
+  '\
+But real speed is ' +
+  `${frameworks[frameworks.length - 1]}`;
+
+//console.log(x);
+
+let y = `google released new version ${frameworks[1]}${Math.floor(frameworks[0])} But real speed is ${frameworks[frameworks.length - 1]}`;
```
Comment (OlegLustenko) [inline js-core/homeworks/homework-2/src/main.js]: such construction doesn't seems pretty clear, can we improve it ?
```js
${frameworks[1]}${Math.floor(frameworks[0])}
```

Comment (OlegLustenko) [reply js-core/homeworks/homework-2/src/main.js]: same here
```js
${frameworks[frameworks.length - 1]}
```

Comment (viskosa) [reply js-core/homeworks/homework-2/src/main.js]: I can't understand what should I do with it. It's about syntax?

Comment (OlegLustenko) [reply js-core/homeworks/homework-2/src/main.js]: > I can't understand what should I do with it. Is it about syntax?

it about readability, for example you can create additional variables or so

Comment (viskosa) [reply js-core/homeworks/homework-2/src/main.js]: something like that? I've fixed

**Review by OlegLustenko — APPROVED** [review]

**Pull request merged**